apps/web-connect/src/features/checkout/Checkout.tsx

```diff
@@ -46,9 +46,9 @@ export function Checkout() {
     }
 
 	useEffect(() => {
-		if (!stakingTabOpened && mintWithEth.isSuccess || mintWithXai.isSuccess) {	
+		if (!stakingTabOpened && (mintWithEth.isSuccess || mintWithXai.isSuccess)) {	
             setStakingTabOpened(true);
-			window.open("https://app.xai.games/staking?mint=true", '_blank');
```

Comment:
After testing the modal logic on the staking app I updated the link to the correct params for not triggering another refresh becasue of missing query params. This would have to be updated / fixed in the staking app